### ISSUE TITLE
chore: Handle deprecation notices for goreleaser

### ISSUE THREAD

**Issue by kevdowney** [keikoproj/addon-manager#187]
**Is this a BUG REPORT or FEATURE REQUEST?**:
CHORE
**What happened**:
```
• DEPRECATED: `archives.replacements` should not be used anymore, check https://goreleaser.com/deprecations#archivesreplacements for more info
```

**What you expected to happen**:
Cleanup [.goreleaser.yml](https://github.com/keikoproj/addon-manager/blob/af4c63c20fec4ee612ab0b437dc95f81ab6aa9af/.goreleaser.yml#L60-L65) to address Goreleaser v4 deprecrations
https://goreleaser.com/deprecations/#archivesreplacements

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Addon Manager version
- Kubernetes version :
```
$ kubectl version -o yaml
```

**Other debugging information (if applicable)**:
- Addon status:
```
$ kubectl describe addon <addon-name>
```
- controller logs:
```
$ kubectl logs <addon-manager-pod>
```

**Comment by kevdowney** [comment]
Hi @anushkabishnoi thanks for taking an interest, above in the Description section **"What you expect to happen"** I've put all the relevant links. We just need to remove the `replacements` section of the [.goreleaser.yml](https://github.com/keikoproj/addon-manager/blob/af4c63c20fec4ee612ab0b437dc95f81ab6aa9af/.goreleaser.yml#L60-L65) file and replace it w/ suitable go template an example of the in the 2nd link in that description, _After_ tab.

**Comment by anushkabishnoi** [comment]
Okay I've created a fork and comitted changes there. Please do check it, and tell me if there's any changes I should make. (and created a pull request too)

**Comment by kevdowney** [comment]
Fixed by #190
